luminance/src/blending.rs

```diff
@@ -76,3 +76,27 @@ pub struct Blending {
   /// Destination factor.
   pub dst: Factor,
 }
+
+/// Blending configuration to represent combined or separate options.
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum BlendOption {
+  /// blending with combined RGBA
+  Combined(Blending),
+
+  /// blending with RGB and alpha separately
+  Separate(Blending, Blending),
+}
+
+impl Into<BlendOption> for Blending {
+  fn into(self) -> BlendOption {
+    BlendOption::Combined(self)
+  }
+}
+
+impl Into<BlendOption> for (Blending, Blending) {
```

Comment:
Hm, I don’t like that implementor. I’m not sure it’s useful at all, and it might be confusing regarding the order each blending setup applies to (is it rgb / alpha or alpha / rgb, etc.).